### DISCUSSION TITLE
Insane Perfomance in Zen3

### DISCUSSION THREAD

**Discussion by Gigantua** [animetosho/md5-optimisation#6] [Q&A]
Ryzen 5950X:

Std   :    959.2 MB/s
GOpt  :   1013.8 MB/s
GHOpt :   1008.3 MB/s
NoLEA : 246414.5 MB/s
NoL-G :   1010.2 MB/s
NoL-GH:   1016.2 MB/s
Cache4:    958.8 MB/s
Cache8:    958.2 MB/s
Cach8G:   1012.6 MB/s

NoLEA: 246414MB/s
Can someone verify? All tests are passing.

**Comment by animetosho** [comment]
Yeah, that's definitely wrong.

What compiler+version are you using, and are you able to upload the compiled binary?
